src/main/java/com/hibernate/model/MovieSession.java

```diff
@@ -6,16 +6,21 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
+import javax.persistence.Table;
 
 @Entity
+@Table(name = "movie_sessions")
 public class MovieSession {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "movie_id")
     private Movie movie;
     @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "hall_id")
     private CinemaHall cinemaHall;
     private LocalDateTime showTime;
```

Comment:
let's also add `@JoinColumn(name = "show_time") for the showTime field

Reply:
How can i do it without relations?

Reply:
you are right - I misspelled, you should better use `@Column` for this purpose if you do not have relations